plugins/logitech-hidpp/fu-logitech-hidpp-bootloader.c

```diff
@@ -342,18 +343,11 @@ fu_logitech_hidpp_bootloader_request (FuLogitechHidPpBootloader *self,
 				    buf_request, sizeof (buf_request));
 	}
 	if (usb_device != NULL) {
-		if (!g_usb_device_control_transfer (usb_device,
-						    G_USB_DEVICE_DIRECTION_HOST_TO_DEVICE,
-						    G_USB_DEVICE_REQUEST_TYPE_CLASS,
-						    G_USB_DEVICE_RECIPIENT_INTERFACE,
-						    FU_HID_REPORT_SET,
-						    0x0200, 0x0000,
-						    buf_request,
-						    sizeof (buf_request),
-						    &actual_length,
-						    FU_UNIFYING_DEVICE_TIMEOUT_MS,
-						    NULL,
-						    error)) {
+		if (!fu_hid_device_set_report (FU_HID_DEVICE (self), 0x0,
```

Comment:
houldn't this also be 0x200?

Reply:
The wValue is being set in fu_hid_device_set_report - so 0x0200 is `FU_HID_REPORT_TYPE_OUTPUT << 8`

Reply:
oh... I completely missed that at the top of the function.  OK never mind, I guess most of my comments are wrong then :)